components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/internal/FrameworkServiceComponent.java

```diff
@@ -462,6 +464,35 @@ protected void setAuthenticator(ApplicationAuthenticator authenticator) {
         }
     }
 
+    @Reference(
+            name = "session.serializer",
+            service = SessionSerializer.class,
+            cardinality = ReferenceCardinality.MULTIPLE,
+            policy = ReferencePolicy.DYNAMIC,
+            unbind = "unsetSessionSerializer"
+    )
+    protected void setSessionSerializer(SessionSerializer sessionSerializer) {
```

Comment:
How the default JavaSessionSerializer is wired when there are no custom serializers are registered?